interface/patient_file/summary/immunizations.php

```diff
@@ -619,53 +619,53 @@ function saveImmunizationObservationResults($id, $immunizationdata)
                                                 <br>
                                                 <select class="form-control" id="observation_criteria_1" name="observation_criteria[]" onchange="selectCriteria(this.id,this.value);">
                                                 <?php foreach ($observation_criteria as $keyo => $valo) { ?>
-                                                    <option value="<?php echo attr($valo['option_id']);?>" <?php echo ($valo['option_id'] == $value['imo_criteria'] && $id != 0) ? 'selected = "selected"' : ''; ?> ><?php echo text($valo['title']);?></option>
```

Comment:
I am pretty sure that $value is meant to be there. Fixing this section of code was a painful thing to fix in PHP8 btw :) . I think i got it fixed up in the ongoing PR here:
#4087